src/components/config_parser.cpp

```diff
@@ -292,4 +294,23 @@ bool config_parser::is_valid_name(const string& name) {
   return true;
 }
 
+string config_parser::parse_escaped_value(string&& value, const string& key) {
+  string cfg_value = value;
+  bool log = false;
+  auto backslash_pos = value.find_first_of('\\');
+  while (backslash_pos != string::npos) {
+    if (value[backslash_pos + 1] != '\\') {
```

Comment:
This can still lead to an out-of-bounds access. Maybe this is the cause of the segfault.

The special case happens only if there is an unescaped backslash at the end of the line, the behavior should be the same as if there was an unescaped backslash anywhere else.

Reply:
Could you please explain why?
I added a fix if I understand what you meant, but the segfaults are still there.

Reply:
You're change is correct. Also, this was not the cause of the segfault.